### PR TITLE
py3 validate

### PR DESCRIPTION
**Pull request by zippolyte**

### CODE REVIEW

File: shared/__init__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import os
+import sys
+
+from .validate import py3
+
+
+def main():
+    """Run validate on a single python file"""
+    if len(sys.argv) != 2:
+        print("No python file specified")
```
Comment (masci) [inline shared/__init__.py]: let's use `sys.stderr.write`

File: shared/validate.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+from pylint.lint import PyLinter, fix_import_path
+from pylint.reporters.json import JSONReporter
+
+
+def py3(path_to_module):
```
Comment (masci) [inline shared/validate.py]: let's make the name more verbose, like `validate_py3`

**Review by gzussa — CHANGES_REQUESTED** [review]
Great reactivity @zippolyte !!
Let's add a description to explain from where this code is originally coming from.
I am not sure if we had unit tests for this code before, but if we could have some here, it would be great. Especially since this is going to be used in different places now.
Regarding CI, can we setup Travis for this project as well?

**Review by gzussa — APPROVED** [review]
We can make Tests and CI config in another PR. Approving but let's address other comments regarding the code implementation itself.

File: a7/__init__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+import os
+import sys
+
+from .validate import validate_py3
+
+
+def main():
+    """Run validate on a single python file"""
+    if len(sys.argv) != 2:
```
Comment (ofek) [inline a7/__init__.py:13]: We shouldn't do this. There may very well be other things this package does so let's avoid extra Agent and `ddev` changes in future and make an arg before to specify action.

**Review by ofek — CHANGES_REQUESTED** [review]

**Pull request merged**